sdks/python/apache_beam/io/gcp/bigquery_schema_tools.py

```diff
@@ -38,8 +39,10 @@
     "STRING": str,
     "INTEGER": np.int64,
     "FLOAT64": np.float64,
+    "FLOAT": np.float64,
     "BOOLEAN": bool,
     "BYTES": bytes,
+    "TIMESTAMP": apache_beam.utils.timestamp.Timestamp.from_utc_datetime
```

Comment:
Should the type here be `apache_beam.utils.timestamp.Timestamp` ?

Reply:
cc: @Abacn @TheNeuralBit who worked on  https://github.com/apache/beam/blob/master/sdks/python/apache_beam/typehints/schemas.py#L36-L37 

Folks, do you know what (if any) aspects we need to take care of for this change to correctly work with schemas & xlang?

Reply:
I had some discussion with @svetakvsundhar before on the question whether implement TIMESTAMP as Python's native `datetime.datetime` or beam's `apache_beam.utils.timestamp.Timestamp`. Both should work within the Python SDK, but `datetime.datetime` is not portable. So I suggested the latter. In a xlang scenario it should just work.

Reply:
Thanks. It seems that we should drop `.from_utc_datetime` as that is a method, which returns the Timestamp.
Any concerns with that @svetakvsundhar ?

Reply:
+1

```suggestion
    "TIMESTAMP": apache_beam.utils.timestamp.Timestamp
```

should do it